src/Metadata/Property/PropertyMetadata.php

```diff
@@ -43,6 +44,7 @@ public function __construct(Type $type = null, string $description = null, bool
         $this->identifier = $identifier;
         $this->iri = $iri;
         $this->attributes = $attributes;
+        $this->childInherited = null;
```

Comment:
Ok, and we're adding the property before `$attributes` right? I'll be waiting for this to be good to go before adding this modification as it'll take more modifications and be a mess to review if that's ok.